blocks/api/registration.js

```diff
@@ -87,6 +94,24 @@ export function getUnknownTypeHandler() {
 }
 
 /**
+ * Assigns the default block name
+ *
+ * @param {string} name Block name
+ */
+export function setDefaultBlockName( name ) {
```

Comment:
Maybe `setDefaultBlock( name )` ? Could be confused as "change the name of the block which is the default", not "set the block to be used as the default".